tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -34,12 +34,12 @@ def cards(self):
 
     @property
     def is_cards_view_displayed(self):
```

Comment:
This can be renamed as `is_displayed`. So we can call this function this way: `cards_view.is_displayed`